components/Profile/ProfileStatsBox.tsx

```diff
@@ -44,9 +36,10 @@ const ProfileStatsBox = ({
             <View className="h-16 w-24 flex-1 flex-col items-center rounded-lg bg-badgesGreen py-2">
               <Image source={require("../../assets/images/profileFlame.png")} />
               <Text
-                className="font-sans-bold mt-2 text-projectWhite"
+                className="text-caption-lg-bold mt-2"
```

Comment:
This doesn't seem to be a preset in the typography Figma. Possibly we removed this preset when we removed your changes in `theme/typography.ts`